redis/cluster.py

```diff
@@ -1274,6 +1291,12 @@ def create_redis_node(self, host, port, **kwargs):
             r = Redis(host=host, port=port, **kwargs)
         return r
 
+    def _is_cluster_enabled(self, redis_node):
```

Comment:
Since this is called only once, it belongs in the initialize function. There's no benefit to separating out this check into a function unless there is reuse.

Reply:
removed